tests/buyProducts_test.js

```diff
@@ -19,34 +19,40 @@ console.log(productLinks2);
 
 Feature('buy product');
 
-Before(({I}) => {
+Before(async ({I, homePage}) => {
     I.login(loginUser);
+    homePage.clickCartIcon();
+    console.log(await homePage.checkCartIsEmpty());
+    await homePage.emptyCart();
 });
 
 Data(productLinks2).Scenario('buy product', async ({ I, productPage, checkoutPage, orderHistoryPage, current }) => {
-    I.see('My Account');
+    
     I.amOnPage(current.link);
     let price = await productPage.getProductPrice();
     console.log(price);
```

Comment:
remove console.logs or add some info to understand where they are coming from. E.g. ('Price is: ' + price). Otherwise, it is just set of numbers in log.

Reply:
Apply for all `console.log()`

Reply:
Done.